app/src/main/java/com/keylesspalace/tusky/entity/Account.kt

```diff
@@ -0,0 +1,67 @@
+/* Copyright 2017 Andrew Dawson
+ *
+ * This file is a part of Tusky.
+ *
+ * This program is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Tusky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+ * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Tusky; if not,
+ * see <http://www.gnu.org/licenses>. */
+
+package com.keylesspalace.tusky.entity
+
+import android.os.Parcel
+import android.os.Parcelable
+import android.text.Spanned
+
+import com.google.gson.annotations.SerializedName
+import com.keylesspalace.tusky.util.HtmlUtils
+import kotlinx.android.parcel.Parceler
+import kotlinx.android.parcel.Parcelize
+import kotlinx.android.parcel.WriteWith
+
+@Parcelize
+class Account (val id: String,
```

Comment:
Why not data class? You could override `equals()` for it as well

Reply:
oh right, missed that